status-reporter/status_reporter/status_reporter.py

```diff
@@ -105,23 +104,36 @@ def _build_report(self, status):
         return {'reporting_freq': self._current_reporting_freq,
                 'report': status}
 
-    def _update_managers_ips_list(self):
+    def _update_managers_ips_list(self, manager_ip):
+        url = 'https://{}:443/api/v3.1/managers'.format(manager_ip)
```

Comment:
are you sure we need to specify the  port?

Reply:
I'm not sure, but I think we'll change it anyway once we write the API, right?